easybuild/easyconfigs/n/netMHCII/netMHCII-2.3.eb

```diff
@@ -27,6 +25,8 @@ checksums = [
     '8b184b2463b93bbcbb93a7351c6c0bae12d82c7827c451ebc5153b257dd0be86',  # netMHCII-2.3.Linux.tar.gz
     '6bf8af9e2e89d96b8fe9a2752164606417c4583872bdc2e81637f6668dee30a0',  # netMHCII-2.3-data.Linux.tar.gz
 ]
+download_instructions = f"""{name} is proprietary software, but free for academics.
+It can be requested at https://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHCII"""
```

Comment:
Same with data.Linux.tar.gz here.

Reply:
same comments apply here as for `netMHC`

Reply:
f3fb169